src/pages/tug/index.page.js

```diff
@@ -116,9 +117,9 @@ const People = ({
                 </div>
                 <div className={styles.header_content_left_summary}>{description}</div>
                 <div className={styles.header_content_left_summary}>
-                  <Button as={MyLink} href={applyUrl}>
-                    加入 TUG
-                  </Button>
+                  <Link href={applyUrl}>
+                    <Button type="primary">加入 TUG</Button>
+                  </Link>
```

Comment:
The changes you did may work, but to wrap a button with a link seems not to be a usual way. Alternatively, you may use handleRedirect in the link utils as we did on the homepage: https://github.com/pingcap/tug-website/blob/main/src/pages/home/welcome/Welcome.component.js#L42-L45

Reply:
OK! I have optimized it.